android/java/org/chromium/chrome/browser/rewards/tipping/RewardsTippingPanelFragment.java

```diff
@@ -510,12 +512,24 @@ private void init(View view) {
         mRadioTipAmount[1] = view.findViewById(R.id.tipChoice2);
         mRadioTipAmount[2] = view.findViewById(R.id.tipChoice3);
         mRadioTipAmount[3] = view.findViewById(R.id.tipChoiceCustom);
+        if (getCurrentLocale().getLanguage().equals(
+                    Locale.UK.getLanguage())) { // This need to override for this language
+            mRadioTipAmount[3].setText(CUSTOM_UK);
```

Comment:
@sujitacharya2005 what's the reason to make a check for UK only here ?

Reply:
Hi @deeppandya 

For UK language instead of "Custom" it's showing "Customized"
@Miyayes mentioned to update to "Custom" for UK language.

Reply:
in that case, it should handled in translations. instead of making an exception for the specific language, we make sure of the change in translations. @mkarolin wdyt ?

Reply:
I posted in #l10n but I have no idea why the word "custom" is translated as "customised" in UK English. The word "custom" exists in UK English, so it just seems like an overzealous translation to me.

Reply:
Great , 
If it's handled from translations. I will revert this change.